packages/codec/src/error.ts

```diff
@@ -0,0 +1,54 @@
+export const CODEC_OPTIONAL_PATH = "______?______";
```

Comment:
```suggestion
// lc for lumos codec
export const CODEC_OPTIONAL_PATH = "__lc_option__";
```
i think the `____?____` is too magic when debugging